swap/swap_test.go

```diff
@@ -404,7 +395,7 @@ func addBookings(swap *Swap, bookings []booking) {
 func verifyBookings(t *testing.T, swap *Swap, bookings []booking) {
 	t.Helper()
 	expectedBalances := calculateExpectedBalances(swap, bookings)
-	realBalances := swap.balances
+	realBalances, _ := swap.Balances()
```

Comment:
As a rule, we never ignore errors in a test. If we can and it make sense, add a comment saying why.

Reply:
done